tests/syntax/test_compiler.py

```diff
@@ -0,0 +1,25 @@
+from ast import *
+from scenic.syntax.ast import *
+
+from scenic.syntax.compiler import compileScenicAST
+
+
+class TestCompiler:
+    # Special Case
+    def test_ego_assign(self):
+        node, _ = compileScenicAST(EgoAssign(Constant(1)))
+        match node:
+            case Expr(Call(func=Name(id=fn_name), args=args)):
```

Comment:
Can clean this up to `Expr(Call(Name("ego"), [Constant(1)]))` and get rid of the assertions (ditto for the similar tests below and in `test_parser.py`). For nodes with many fields it might be clearer to keep the names of the arguments (e.g. `Call(func=Name("ego"), args=[Constant(1)])`), but for nodes like these I don't think omitting the names will cause confusion.

Reply:
fe04f4a732c0e8346076705d3359a3f864b500cd

I removed the argument names from `test_compiler.py`. Let me work on the same change for `test_parser.py` on another PR as it requires some work on Scenic AST classes to be able to pattern match without keywords (more specifically, I need to add `__match_args__` to every class).

Reply:
Ah, right. Not a high priority in any case, just something to keep in mind.